lmdeploy/turbomind/turbomind.py

```diff
@@ -329,6 +329,9 @@ def _broadcast_np(data, dtype, shape=(batch_size, )):
             while self.que.qsize() > 1:
                 self.que.get()
 
+            if stream_output == False and self.que.qsize() == 0:
+                yield
```

Comment:
I found this made GPU-Util quite low. And the benchmark went down to an unacceptable number.

Reply:
> I found this made GPU-Util quite low. And the benchmark went down to an unacceptable number.

I find the first version has this problem with streaming mode, 
```python
if  self.que.qsize() == 0:
    yield
```

So I modified it, and I test the GPU util is 100% for both modes:
```python
            if stream_output == False and self.que.qsize() == 0:
                yield
```

Anyway, I followed your advice and use stream infer to process non-stream requests.